src/wallet.js

```diff
@@ -0,0 +1,90 @@
+import { existsSync, readFileSync, writeFileSync } from 'fs'; // Import the fs module for file operations
+import { GetMempool } from './Mempool';
+import { Transaction, UnsignedTransaction } from './Transaction';
+
+
+class Wallet {
+    constructor() {
+        if (existsSync('private_key.json')) {

```

Comment:
You may get an error here while reading the file, example when you don't have permissions. Can we add error handling?